changelog

```diff
@@ -1,3 +1,7 @@
+2017-12-06, version 1.3.2
+    * Correctly detect and handle socket disconnection on Windows
+    * Handle more concurrent streams on windows
```

Comment:
More than what? I suggest to rename "handle an arbitrary number of concurrent streams on Windows"